### PR TITLE
DTSSTCI-354

### PR DESCRIPTION
**Pull request by gmmagruder**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DTSSTCI-354

### Change description ###
Update check your answers page
Screenshot 2024-03-06 at 11 39 28

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/steps/dss-update/check-your-answers/content.test.ts [hunk 1]
```diff
@@ -5,19 +5,17 @@ import { CommonContent, generatePageContent } from '../../common/common.content'
...
   submitApplicationText: 'Now submit your Application',
-  listOfDocuments: 'List of documents uploaded',
   errorSummaryMessage: 'There is a problem',
-  declarationText:
-    'I understand that proceedings for contempt of court may be brought against anyone who makes, or causes to be made, a false statement in a document verified by a statement of truth without an honest beleif in its truth.',
   keys: {
-    fileName: 'File name',
-    description: 'Description',
+    information: 'Additional information',
+    fileName: 'Additional documents',
```
Comment (rajaram-nagarajan) [inline src/main/steps/dss-update/check-your-answers/content.test.ts:17]: Better to change the left hand side name to match with the right hand side description?
Example: fileName: 'Additional documents' can be changed to 
documents: 'Additional documents' or something

**Review by tomxelliott — DISMISSED** [review]

**Review by oleon23 — DISMISSED** [review]

File: src/main/app/controller/AppRequest.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ export interface AppSession extends Session {
   lang: string | undefined;
   errors: FormError[] | undefined;
   addresses: [];
+  fileErrors: any[];
```
Comment (gmmagruder) [inline src/main/app/controller/AppRequest.ts:27]: From DTSSTCI 353 (https://github.com/hmcts/sptribs-dss-update-case-web/pull/35)

File: src/main/app/controller/GetController.ts [hunk 3]
```diff
@@ -68,12 +68,16 @@ export class GetController {
...
+        fileErrors: req.session.hasOwnProperty('fileErrors') ? req.session.fileErrors : [],
         caseId: req.session.userCase?.id,
       };
 
       if (req.session?.errors) {
         req.session.errors = undefined;
       }
+      if (req.session?.fileErrors) {
+        req.session.fileErrors = [];
+      }
```
Comment (gmmagruder) [inline src/main/app/controller/GetController.ts:80]: From DTSSTCI 353 (https://github.com/hmcts/sptribs-dss-update-case-web/pull/35)

File: src/main/steps/upload-documents/content.test.ts [hunk 4]
```diff
@@ -1,47 +1,49 @@
-import { cy, en, form, generateContent } from './content';
+import { cy, en, form, generateContent, getErrors } from './content';
```
Comment (gmmagruder) [inline src/main/steps/upload-documents/content.test.ts:1]: All file changes in /upload-documents are from DTSSTCI 353 (https://github.com/hmcts/sptribs-dss-update-case-web/pull/35)

**Review by tomxelliott — APPROVED** [review]

**Review by oleon23 — DISMISSED** [review]

**Review by tomxelliott — DISMISSED** [review]

**Review by oleon23 — APPROVED** [review]

**Review by tomxelliott — DISMISSED** [review]

**Review by oleon23 — DISMISSED** [review]

**Review by oleon23 — APPROVED** [review]

**Review by tomxelliott — APPROVED** [review]

**Pull request merged**